orte/orted/pmix/pmix_server.c

```diff
@@ -514,6 +515,7 @@ static void pmix_server_dmdx_recv(int status, orte_process_name_t* sender,
         ORTE_ADJUST_TIMEOUT(req);
         if (OPAL_SUCCESS != (rc = opal_hotel_checkin(&orte_pmix_server_globals.reqs, req, &req->room_num))) {
             orte_show_help("help-orted.txt", "noroom", true, req->operation, orte_pmix_server_globals.num_rooms);
+            ORTE_ERROR_LOG(rc);
```

Comment:
We usually don't error log after showing help as it just confuses the user. Is there some reason to be doing this? If you are just trying to log the location, then perhaps you should just add that to the show_help message.

Reply:
It may have been for debugging purposes. @jjhursey is there a reason to keep these logging messages?

Reply:
IIRC we were trying to debug why the `opal_hotel_checkin` failed so we added these for developer debugging purposes. I'm fine if we want to drop them.

Reply:
Done - removed. Thanks